src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -161,7 +161,7 @@ public static void UpdateBackgroundLayerFrame(this UIView view)
 		public static void InvalidateMeasure(this UIView nativeView, IView view)
 		{
 			nativeView.SetNeedsLayout();
-			nativeView.Superview.SetNeedsLayout();
+			nativeView.Superview?.SetNeedsLayout();
```

Comment:
May not have a superview yet. You can't actually launch iOS without this fix. And the CI tests failed with an exception.